src/components/description-list/_description-list.scss

```
@@ -7,25 +7,61 @@
         clear: both;
         float: left;
         font-weight: $font-weight-bold;
-
-        &:not(:first-child) {
-            margin-top: 0.5rem;
-        }
     }
 
     &__value {
         float: right;
         margin-left: 0; /* As normalize adds a 40px left margin */
+    }
 
-        &:not(:nth-of-type(1)) {
-            @include mq(m) {
+    &__item {
+        &:not(:first-child) {
+            .ons-description-list__term {
                 margin-top: 0.5rem;
             }
+
+            .ons-description-list__value:nth-of-type(1) {
+                @include mq(m) {
+                    margin-top: 0.5rem;
+                }
+            }
+
+            .ons-description-list--inline & {
+                .ons-description-list__value:nth-of-type(1) {
+                    @include mq(l) {
+                        margin-top: 0;
+                    }
+                    @include mq(xs, l) {
+                        margin-top: 0.5rem;
+                    }
+                }
+
+                .ons-description-list__term {
+                    @include mq(l) {
+                        margin-top: 0;
+                    }
+                }
+            }
+        }
+    }
+
+    &--inline {
```

```suggestion
    &--inline & {
```
This can be changed like this and then you can replace all the instances of ".ons-description-list" below with "&"